packages/tracing-internal/src/browser/request.ts

```diff
@@ -166,6 +166,15 @@ export function instrumentOutgoingRequests(_options?: Partial<RequestInstrumenta
   }
 }
 
+function isPerformanceResourceTiming(entry: PerformanceEntry): entry is PerformanceResourceTiming {
+  return (
+    entry.entryType === 'resource' &&
+    'initiatorType' in entry &&
+    'nextHopProtocol' in entry &&
```

Comment:
l: this still doesn't guarantee that `nextHopProtocol` is a string - at the same time checking whether the key exists is probably enough

Reply:
yeah, I figured the same.. I mean these are browser APIs, I think _if_ this exists it will be a string, this is more to make sure this is not some other type of resource... IMHO we can do

1. Just check existence (current code)
2. Check existence + `typeof entry.nextHopProtocol === 'string`
3. Only check type - requires type cast (which is fine I guess) `typeof (entry as PerformanceResourceTiming).nextHopProtocol === 'string`

Reply:
I don't mind either way